x-pack/legacy/plugins/lens/public/indexpattern_plugin/dimension_panel/popover_editor.tsx

```diff
@@ -196,26 +197,46 @@ export function PopoverEditor(props: PopoverEditorProps) {
     <EuiPopover
       id={columnId}
       className="lnsConfigPanel__summaryPopover"
-      anchorClassName="lnsConfigPanel__summaryPopoverAnchor"
+      anchorClassName={
+        selectedColumn ? 'lnsConfigPanel__summaryPopoverAnchor' : 'lnsConfigPanel__summaryLink'
+      }
       button={
         selectedColumn ? (
           <EuiLink
             className="lnsConfigPanel__summaryLink"
             onClick={() => {
-              setPopoverOpen(true);
+              setPopoverOpen(isPopoverOpen ? false : true);
```

Comment:
This should just be: `setPopoverOpen(!isPopoverOpen)` Also, I think Tim had a PR out that did this, so you may end up with merge conflicts if he merges that, just FYI.

Reply:
Or **he** will have merge conflicts 😆

Reply:
```suggestion
              setPopoverOpen(!isPopoverOpen);
```